src/zkevm_llvm/main.rs

```diff
@@ -4,7 +4,9 @@
 
 pub(crate) mod arguments;
 
+use compiler_llvm_builder::utils;
 use std::path::PathBuf;
+use utils::check_presence;
```

Comment:
1. Here you're importing an item using 2 'linked' imports, instead of one.
2. Please completely avoid imports for functions of the same crate and instead call them directly. It improves readability as you can see what is actually called in-place.

Reply:
Fixed